src/api/middleware/company.js

```diff
@@ -1,5 +1,7 @@
 const { ErrorTypes } = require("./errorHandler");
 const HTTPStatus = require("http-status-codes");
+const CompanyService = require("../../services/company");
+const CompanyConstants = require("../../models/constants/Company");
 
 const isCompanyRep = (req, res, next) => {
     if (!req.user.company) {
```

Comment:
Let's use optional chaining after #77 kicks in

```if (!req.user?.company)```

To avoid errors when req.user is not set.